deepinv/tests/test_physics.py

```diff
@@ -167,12 +168,42 @@ def test_operators_adjointness(name, device):
     ):  # automatic adjoint does not work for inputs that are not torch.tensors
         return
     f = adjoint_function(physics.A, x.shape, x.device, x.dtype)
+
     y = physics.A(x)
     error2 = (f(y) - physics.A_adjoint(y)).flatten().mean().abs()
 
     assert error2 < 1e-3
 
 
+@pytest.mark.parametrize("name", PADDING_OPERATORS)
+@pytest.mark.parametrize("padding", BOUNDARY)
+def test_operator_with_boundary(name, device, padding):
+    torch.manual_seed(0)
+
+    physics, imsize, norm_ref, dtype = find_operator(name, device, padding=padding)
+
+    # Test adjointess of the implemented adjoint function
+    x = torch.randn(imsize, device=device, dtype=dtype).unsqueeze(0)
+    error = physics.adjointness_test(x).abs()
+    assert error < 1e-3
+
+    # Test adjointess of the adjoint function given by auto diff
+    f = adjoint_function(physics.A, x.shape, x.device)
+    y = physics.A(x)
+    error2 = (f(y) - physics.A_adjoint(y)).flatten().mean().abs()
+    assert error2 < 1e-3
+
+    # Test operator norm
+    # norm = physics.compute_norm(x)
+    # assert torch.abs(norm - norm_ref) < 0.2
```

Comment:
It was for review, I think it can be deleted. To quote Pierre (on the Idris Hackaton Slack): 
> In general spectral norms of convolution operators are different from 1 (except for filters on the simplex with periodic boundary conditions)...

So we chose not to test the norm for all boundary conditions.